Trial Management hub [DOCS-12956] (#34770)
DataDog/documentation: content/en/account_management/plan_and_usage/_index.md

Your account billing history is available from the **Billing History** tab in the [Plan & Usage section][2]. For specific billing questions, contact your account representative or [success@datadoghq.com][4].

## Trial management

> If your organization has access to product trials, users can manage their trials from the **Trial Management** tab in the [Plan & Usage section][2]. Product trials allow you to try out new products for a limited time without incurring costs. There will be no charge during the free trial period, but you will be charged for any post-trial usage.

Reviewer: ```suggestion
If your organization has access to product trials, users can manage their trials from the **Trial Management** tab in the [Plan & Usage section][2]. Product trials let you try out new products for a limited time without incurring costs. There is no charge during the free trial period, but you are charged for any post-trial usage.
```
